services/github/github-hacktoberfest.service.js

```diff
@@ -65,8 +65,8 @@ module.exports = class GithubHacktoberfestCombinedStatus extends GithubAuthV4Ser
 
   static get route() {
     return {
-      base: 'github/hacktoberfest/2019',
-      pattern: ':user/:repo',
+      base: 'github/hacktoberfest',
+      pattern: ':year(2019|2020)/:user/:repo',
```

Comment:
As discussed when we first added this badge last year, [we want to keep the year as an explicit part of the route](https://github.com/badges/shields/pull/4109#issuecomment-537642828). I wasn't sure if we wanted to completely open this up just yet, or keep it constrained to a couple years, but obviously opted for the latter.